peachjam/signals.py

```diff
@@ -0,0 +1,41 @@
+import sentry_sdk
```

Comment:
Is this file imported automatically, or does it need to be imported in an `app.ready()` somewhere?

Reply:
Added the import